src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -16,8 +16,9 @@
 import os
 import sas
 import sas_testcase
-from util import winnforum_testcase, getRandomLatLongInPolygon, \
-  makePpaAndPalRecordsConsistent
+import logging
+from util import winnforum_testcase, writeConfig,loadConfig, configurable_testcase ,getRandomLatLongInPolygon, \
```

Comment:
Maintain the same order as in master. Let the new functions that we added be in a separate line.